ext/opentelemetry-ext-flask/src/opentelemetry/ext/flask/__init__.py

```diff
@@ -1,9 +1,23 @@
-# Note: This package is not named "flask" because of
-# https://github.com/PyCQA/pylint/issues/2648
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# See https://github.com/PyCQA/pylint/issues/2648
+# pylint: disable=import-self
 
 import logging
 
-from flask import request as flask_request
+from flask import request as flask_request  # pylint: disable=no-name-in-module
```

Comment:
Even when suppressing this error, it also means that pylint won't be able to do any checks whether some attribute exists on flask_request. But given that I also really don't like ` flask_util` as package name, I'm inclined to accept that shortcoming.